sys-devel/parity/parity-9999.ebuild

```diff
@@ -8,11 +8,11 @@ if [[ ${PV} == 9999 ]]; then
 	EGIT_REPO_URI="git@github.com:ssi-schaefer/parity.git https://github.com/ssi-schaefer/parity.git"
 	BDEPEND="dev-util/confix"
 else
-	SRC_URI="https://github.com/mduft/${PN}/releases/download/${PV}/${P}.tar.bz2"
+	SRC_URI="https://github.com/ssi-schaefer/${PN}/releases/download/${PV}/${P}.tar.bz2"
```

Comment:
Let's just drop the 9999 ebuild here, it depends on `dev-util/confix` which does not exist.